various-receipes/authentication/test/auth-example-with-token.test.js

```diff
@@ -0,0 +1,95 @@
+const request = require("supertest");
+const sinon = require("sinon");
+const {
+    initializeWebServer,
+    stopWebServer
+} = require("../api-extension");
+const nock = require("nock");
+const signTokenSynchronously = require("./helper");
+
+let expressApp;
+let sinonSandbox;
+let defaultValidToken;
+
+beforeAll(async (done) => {
+    defaultValidToken = signTokenSynchronously('test-user', 'user', Math.floor(Date.now() / 1000) + (60 * 3600));
```

Comment:
I think the last part is really confusing and we need to encapsulate it in the helper.
I think this would be better:

```suggestion
    defaultValidToken = getValidToken();
```

Reply:
You're right, changed